rest/data/host_create_test.go

```diff
@@ -146,4 +146,52 @@ buildvariants:
 		assert.NotEmpty(settings["key_name"])
 		assert.InDelta(time.Now().Add(cloud.DefaultSpawnHostExpiration).Unix(), h.ExpirationTime.Unix(), float64(1*time.Millisecond))
 	}
+
+	// test that a host.create in a function works too
+	assert.NoError(db.ClearCollections(host.Collection))
+	versionYaml = `
+functions:
```

Comment:
This test should probably include a host.create command not in a function.

Reply:
The test above does this